standard/statements.md

```diff
@@ -361,6 +361,7 @@ expression_statement
 
 statement_expression
     : invocation_expression
+    | null_conditional_invocation_expression
```

Comment:
Again, we may be able to remove this... I'm not sure. (I suspect I need to look at invocation_expression more closely.)

Reply:
Agree